service/create/pki.go

```diff
@@ -0,0 +1,119 @@
+package create
+
+import (
+	"strings"
+
+	microerror "github.com/giantswarm/microkit/error"
+
+	"github.com/giantswarm/certctl/service/pki"
+	"github.com/giantswarm/certctl/service/token"
+)
+
+const (
+	// TODO Choose a sensible value and move to ConfigMap for V1
+	CertificateAuthorityTTL = "1440h"
+)
+
+func (s *Service) checkPKIBackend(clusterID string) bool {
```

Comment:
I also think that it would make sense to put this logic into the certctl service and use it here. This is a common functionality we otherwise copy/paste in case it is required in other places.